python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -521,6 +530,27 @@ def __call__(self, quantum: Quantum) -> LimitedButler:
             dataset_types=self.dataset_types,
         )
 
+    @classmethod
+    def _unpickle(
+        cls, butler_config: Config, dimensions_config: DimensionConfig | None, dataset_types_pickle: bytes
+    ) -> _QBBFactory:
+        universe = DimensionUniverse(dimensions_config)
+        dataset_types = pickle.loads(dataset_types_pickle)
+        return _QBBFactory(butler_config, universe, dataset_types)
+
+    def __reduce__(self) -> tuple:
+        # If dimension universe is not default one, we need to dump/restore
+        # its config.
+        config = self.dimensions.dimensionConfig
+        default = DimensionConfig()
+        if (config["namespace"], config["version"]) != (default["namespace"], default["version"]):
```

Comment:
Maybe add a comment saying that you don't need to send the config in the pickle if the version matches the default universe?